### PR TITLE
Fix #39: Time controls are broken on city change

### LINKED ISSUE #39: Time-selection controls don't update correctly upon city change
The time-selection radio buttons in the legend don't update correctly when the city is changed. In particular, if you switch from a city where there's no market today (i.e. where only the "alle" button is enabled) to a city which does have markets today then the other radio buttons are not re-enabled.

### PR DESCRIPTION
**Pull request by torfsen**
Fixes #39.

The easiest way to test this is to fix date and time to a Monday shortly before 20:00:

* Line 21: `var now = Date("2016-04-18T19:59:00");
* Line 151: `var monday = moment("2016-04-18T19:59:00").startOf("week").add(1, 'days').toDate();`
* Line 152: `var sunday = moment("2016-04-18T19:59:00").endOf("week").add(1, 'days').toDate();`

You can then switch between Karlsruhe (which has a market that is open on Mondays at that time), Erlangen (which has a market that is open on Mondays but not on that time) and Paderborn (which doesn't have an open market on Mondays). The time controls should reflect that: the right radio buttons should be enabled/disabled and the most-specific enabled radio button should be auto-selected.

### CODE REVIEW

**Comment by dirkschumacher** [comment]
Tested it. Works fine as far as I can see it. :shipit:

File: js/main.js [hunk 1]
```diff
@@ -377,6 +366,7 @@ $(document).ready(function() {
     var legend = L.control({position: 'bottomright'});
     var dropDownCitySelection = $('#dropDownCitySelection');
     legend.onAdd = function () { return L.DomUtil.get('legend'); };
+    $("input[name=display]").change(updateLayers);
```
Comment (dirkschumacher) [inline js/main.js]: Can you use single quotes? I think this seems to become our emerging coding style :)

**Comment by johnjohndoe** [comment]
@torfsen I rebased your branch on top of `master` so it's easier to review it.

File: js/main.js [hunk 2]
```diff
@@ -89,44 +89,35 @@ function positionMap(mapInitialization) {
...
-            $('#today').attr('checked', true);
-        } else {
-            $('#other').attr('checked', true);
-        }
-    }
-    $("input[name=display]").change(updateLayers);
+function updateControls() {
+    var gotNow = nowGroup.getLayers().length > 0;
+    var gotToday = todayGroup.getLayers().length > 0;
+    $('#now').prop('disabled', !gotNow);
```
Comment (johnjohndoe) [inline js/main.js]: Nice hack.

**Pull request merged**